<?xml version="1.0" encoding="utf-8"?>
<report>
	<activity
		result="Failed"
		duration="0ms"
		durationms="0"
		type="root"
		rid="c738ef7872ade48"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		user="automation"
		host="DESKTOP-IP0304R"
		rxversion="5.4.0.24580"
		osversion="Windows 10 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		language="en-US"
		screenresolution="1024x768"
		timestamp="3/10/2016 1:15:34 PM"
		timestampiso="2016-03-10T18:15:34"
		timeoutfactor="1"
		module="DisplayQAJobs">
		<detail>
			DisplayQAJobs
		</detail>
		<varbindings/>
		<item
			timeRelativeToTestSuiteStartTime="00:06.436"
			timeRelativeToTestModuleStartTime="00:06.436"
			timeWallClock="13:15:41"
			time="00:06.436"
			level="Info"
			category="User">
			<message>
				Number of links (24) is equal to rangeTo (24)
			</message>
			<metainfo
				codefile="c:\Users\automation\Documents\RanorexSession2_Cl\Session2_CL\Session2_CL\DisplayQAJobs.cs"
				codeline="70"
				loglvl="Info"/>
		</item>
		<item
			timeRelativeToTestSuiteStartTime="00:22.077"
			timeRelativeToTestModuleStartTime="00:22.077"
			timeWallClock="13:15:56"
			time="00:22.077"
			level="Error"
			category="Module">
			<message>
				Test run was aborted by the user.
			</message>
			<metainfo
				loglvl="Error"/>
		</item>
	</activity>
</report>
